src/pages/billing/NewBillingProjectWizard/StepWizard/StepWizard.ts

```diff
@@ -0,0 +1,18 @@
+import { div, h2 } from 'react-hyperscript-helpers'
+
+
+export interface StepWizardProps {
```

Comment:
The components for this are mostly just HOC with some default styling and behavior.